spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  let!(:post_0) { create(:post, message: "First post", id: 0) }
```

Comment:
We need the post messages to be distinct in order to ensure the page ordering is correct, I've removed as much flabby code as possible though